### DISCUSSION TITLE
How to backup and restore postgresql database?

### DISCUSSION THREAD

**Discussion by kingfishx23** [prisma/prisma#9689] [Q&A] [answered]
Im sorry if this is a dumb question, but how do I do this if im using prisma?

any guides on how to do this?

Thanks alot!

**Comment by ryands17** [comment, marked as answer]
@KingfishX23 👋 

Usually you would backup/restore your database on the database provider/platform that you're working on. For e.g. If you're using Heroku or AWS RDS, then you would backup/restore your database from that platform itself as those platforms would have specific settings for those.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
